fedmsg/commands/trigger.py

```diff
@@ -110,11 +110,11 @@ def run(self):
         max_queue_size = int(self.config['max_queue_size'])
 
         timer = None
-        queue = Queue.Queue()
+        que = queue.Queue()
```

Comment:
`que` is not english word, why rename the import (comparing to original code)?
it's ugly if your code reads like you can't spell (do common spellinge errors).

`from six.moves import queue as Queue` afaik works fine